src/document.ts

```diff
@@ -79,16 +79,16 @@ export class Document {
    * Document.list(pod, '/content/pages/**')
    *
    * // Only Markdown docs within the "pages" collection:
-   * Document.list(pod, '/content/pages/*.md')
+   * Document.list(pod, '/content/pages/**\/*.md')
```

Comment:
Does this need to be escaped?

Reply:
It does in the comments otherwise the `*/` ends the block comment.

In the generated docs it does not show up as escaped: https://blinkkcode.github.io/amagaki/classes/pod.pod-1.html#docs

Reply:
Looking into it more, it is a bit of a thorny problem for TSDoc to fix, and I don't know when it will get fixed, but I agree, it is not good for the example to have the escaping.

https://github.com/microsoft/tsdoc/issues/166

Not sure atm what other options we have to show the correct glob pattern to use for the matching. Maybe we could link it to the tests? but that isn't very intuitive to understand.